client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ReachContact.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { TextField, Grid } from "@material-ui/core";
+import { Controller, useFormContext } from "react-hook-form";
+import { Autocomplete } from "@material-ui/lab";
+
+import ContactStatus from "models/ContactStatus";
+import PhoneDial from "commons/PhoneDial/PhoneDial";
+import InteractedContact from "models/InteractedContact";
+import useContactFields from "Utils/vendor/useContactFields";
+import InteractedContactFields from "models/enums/InteractedContact";
+
+
+import useReachContact from "./useReachContact";
+import useStyles from "../ContactQuestioningStyles";
+
+const ReachContact = (props: Props) => {
+    const { control, getValues } = useFormContext();
+    const {
+        interactedContact,
+        index,
+        contactStatuses,
+        saveContact,
+        parsePerson,
+    } = props;
+
+    const classes = useStyles({});
+    const [contactStatusInput, setContactStatusInput] = React.useState<string>(
+        ""
+    );
+
+    const currentValue = contactStatuses.find(
+        (contactStatus: ContactStatus) =>
+            contactStatus.id === interactedContact.contactStatus
+    );
+
+    const formValues = getValues().form
+        ? getValues().form[index]
+        : interactedContact;
+    const { isFieldDisabled } = useContactFields(formValues.contactStatus);
+
+    const { changeContactStatus } = useReachContact({
+        saveContact,
+        parsePerson,
+        formValues,
+        index,
+    });
+
+    return (
+        <div className={classes.reachContact}>
+            <Grid item xs={8} container>
+                <Controller
+                    control={control}
+                    name={`form[${index}].${InteractedContactFields.CONTACT_STATUS}`}
+                    defaultValue={interactedContact.contactStatus}
+                    render={(props) => {
+                        return (
+                            <Autocomplete
+                                {...props}
```

Comment:
yup fixed [**here**](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/964/commits/dfe139bc9bb7ca552de3323d14469a9466705d2a), 
happened because some fields like Id type value for example are different from what the form value should be
(id type is toggle (true/false) and the field is a string (ת"ז or דרכון) ) and I copy-pasted the {...props} from a different input :/